src/envo/envo-edit.owl

```diff
@@ -39742,6 +39745,51 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105011> (water column)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.merriam-webster.com/dictionary/water%20column>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105011> "A field which stretching between the surface and the floor of a body of water."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105011> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105011> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105011> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105011> "2021-05-05T12:26:45Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105011> "Wassersäule"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105011> "colonna d'acqua"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105011> "columna de agua"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105011> "envoPlastics"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105011> "water column"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105011> <http://purl.obolibrary.org/obo/ENVO_01000352>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105012> (biota)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.thefreedictionary.com/biota>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105012> "A material entity which is a group of organisms existing in a region or habitat."@en)
```

Comment:
Another aspect which is mentioned [here](https://www.lexico.com/definition/biota), is that it could also refer to the living organisms of a _geological period_. 

> A material entity which includes the living organisms of a region, habitat, or geological period.

Reply:
This seems like it should be in PCO, but PCO imports NCBI Taxon's [cellular organisms](http://purl.obolibrary.org/obo/NCBITaxon_131567) which has a related synonym "biota". 

I'm not really happy with that, so let's create a new term and post on the PCO tracker asking for its future replacement. 

Place it under PCO's [collection of organisms](http://purl.obolibrary.org/obo/PCO_0000000) and the definition would look like:

> A collection of organisms which is inhabits the same region, ecosystem, or time period. 

I think this is more accurate than using PCO's ecological community, as - in principle - there's no real constraint on how many species need to be present in the biota of some place or time. 

I wonder if "collection of organisms" is enough, however. One thing that lacks is that the collection is interacting with one another (e.g. vs a collection of organisms in a museum)

Reply:
> I wonder if "collection of organisms" is enough, however. One thing that lacks is that the collection is interacting with one another (e.g. vs a collection of organisms in a museum)

@pbuttigieg we think the distinction is already covered by the word "inhabits" (as you wouldn't say the collection of organisms inhabits a museum)

We have requested the addition of those terms on [PCO issue tracker](https://github.com/PopulationAndCommunityOntology/pco/issues/105)